### PR TITLE
feat: batch_labels_runnable

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: batch/create/create_batch_labels_runnable.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      jobId: jobName,
+      job,
+    };
+
+    // Run request
+    const [response] = await batchClient.createJob(request);
+    console.log(JSON.stringify(response));
+  }
+
+  callCreateBatchLabelsRunnable();
```
Comment (subfuzion) [inline batch/create/create_batch_labels_runnable.js:122]: Missing `await`. All of the other files you touched are missing await as well; one of our goals is to refactor to leave code in better shape when we can, so if you don't mind, please fix those also. Thanks!

**Review by subfuzion — CHANGES_REQUESTED** [review]
Thanks for the PR, @gryczj

File: batch/create/create_batch_labels_runnable.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  // Project ID or project number of the Google Cloud project you want to use.
+  const projectId = await batchClient.getProjectId();
+  // Name of the region you want to use to run the job. Regions that are
+  // available for Batch are listed on: https://cloud.google.com/batch/docs/get-started#locations
+  const region = 'europe-central2';
+  // The name of the job that will be created.
+  // It needs to be unique for each project and region pair.
+  const jobName = 'batch-labels-runnable';
+  // Name of the label1 to be applied for your Job.
+  const labelName1 = 'runnable_label_name_1';
```
Comment (msampathkumar) [inline batch/create/create_batch_labels_runnable.js:40]: Hi, quick question. Is it possible to use the same variables as we have in here https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/batch/snippets/src/main/java/com/example/batch/CreateBatchRunnableLabel.java#L17-L142 ?

File: batch/create/create_batch_custom_events.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ async function main() {
   /**
    * TODO(developer): Update these variables before running the sample.
    */
+  // Project ID or project number of the Google Cloud project you want to use.
```
Comment (msampathkumar) [inline batch/create/create_batch_custom_events.js:31]: Thanks for this!

File: batch/create/create_batch_custom_events.js [hunk 4]
```diff
@@ -105,7 +106,7 @@ async function main() {
     console.log(JSON.stringify(response));
   }
 
-  callCreateBatchCustomEvents();
+  await callCreateBatchCustomEvents();
```
Comment (msampathkumar) [inline batch/create/create_batch_custom_events.js:109]: QQ: Will this need any changes in the function code? (In Python, we need to updates the function def use `await`)

Comment (gryczj) [reply batch/create/create_batch_custom_events.js:109]: in this case there is no need to make changes as we already marked this function as asynchronous using keyword: **async** in the declaration

File: batch/create/create_batch_labels_runnable.js (hunk 2 above)
Comment (gryczj) [reply batch/create/create_batch_labels_runnable.js:40]: names adjusted

**Review by subfuzion — APPROVED** [review]

**Pull request merged**